wewatch-client/src/components/Match/Match.js

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from "react";
 import { Redirect } from "react-router-dom";
 
-import MovieDetail from 'views/swiping/movie-detail.js';
+import MatchMovieDetail from './match-movie-detail.js';
```

Comment:
hmm since we made src the main component to import from, we can just import using `'views/swiping/movie-detail.js';` rather than the `./` configuration. Either way is fine, but i think its important to have consistency throughout the application. Let me know your thoughts!

Reply:
I agree! I made the change. Thanks for pointing that out.